src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -32,12 +32,11 @@
 		public static class PWM {
 			public static final int leftDriveMotor = 0;
 			public static final int rightDriveMotor = 1;
-			public static final int ballInnie = 2;
-			// public static final int trayInnie = 3; ----- PROJECT CURRENTLY PAUSED
-			public static final int flywheelLeftMotor = 4;
-			public static final int flywheelRightMotor = 5;
-			public static final int vomitElevator = 6;
-			public static final int vomitOuttakeRoller = 7;
+			public static final int flywheelLeftMotor = 2;
```

Comment:
Make sure all of these ports are correct ones (or heavily annotated to say they are WIP) before merging.